SpatialGDK/Source/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -678,7 +678,16 @@ void USpatialActorChannel::SpatialViewTick()
 	if (Actor != nullptr && !Actor->IsPendingKill() && IsReadyForReplication())
 	{
 		bool bOldNetOwned = bNetOwned;
-		bNetOwned = Actor->GetNetConnection() != nullptr;
+
+		bNetOwned = false;
+		if (UNetConnection* Connection = Actor->GetNetConnection())
+		{
+			if (APlayerController* PlayerController = Connection->PlayerController)
+			{
+				bNetOwned = PlayerController->PlayerState != nullptr;
```

Comment:
Perhaps a comment about the logic for people stumbling upon this code.

Reply:
Whoops 😬

Reply:
I'm too slow. Jetlag!